lib/actions/authActions.js

```diff
@@ -60,3 +53,16 @@ export function signup(newUserData) {
     })
   };
 }
+
+export function validateToken({accessToken, uid, client}) {
+  return {
+    types: [VALIDATE_TOKEN, VALIDATE_TOKEN_SUCCESS, VALIDATE_TOKEN_FAIL],
+    promise: _client => _client.get('/auth/validate_token', {
+      data: {
```

Comment:
@dviramontes this works with data, and I'm not very familiar with how `superagent` works, but should we use params instead? We would just need to handle it in `ApiClient`

Also, why the underscore on `_client`?